### PR TITLE
option to increase console font size

### LINKED ISSUE #14989: Text size controller button in phpmyadmin, SQL.
### Is your feature request related to a problem? Please describe.

Text size is too small. To make it big one has to force zoom, which makes other elements hectic to look.

### Describe the solution you'd like

So to solve this issue, we should introduce a text size controller button.

### Describe alternatives you've considered

Alternative would be giving default font size an appropriate value.

### Additional context

- phpMyAdmin version:4.8.5

### PR DESCRIPTION
**Pull request by bahl24**
Signed-off-by: Nitish Bahl <nitishbahl24@gmail.com>

### Description

Option to change size of text in console
screenshot from 2019-03-06 23-53-20
### Testing Instructions
Go to options tab in console and select console size you want

Fixes #14989 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by bahl24** [comment]
@williamdes I have added option to select size of console, should I add option for 
screenshot from 2019-03-07 00-21-28
or not?
As the feature was removed deliberately earlier?

**Comment by williamdes** [comment]
I think it makes sense to have it also in the SQL query box :)

@ibennetch ?

File: templates/console/display.twig [hunk 1]
```diff
@@ -168,6 +168,16 @@
                     <input type="checkbox" name="dark_theme">{% trans 'Switch to dark theme' %}
                 </label>
                 <br>
+                <label>
+                    Console Size : <select name="console_size" onclick="
+                        $('#pma_console')[0].style.fontSize = $('#pma_console_options').find('select[name=console_size]')[0].value;
```
Comment (shucon) [inline templates/console/display.twig]: It would be better if you remove inline JS and include it in a JS file.

**Review by shucon — CHANGES_REQUESTED** [review]

**Comment by bahl24** [comment]
@shucon @williamdes Kindly check

**Review by shucon — APPROVED** [review]

**Comment by williamdes** [comment]
@sudo-bot Deploy pull-request

**Review by williamdes — CHANGES_REQUESTED** [review]
image

Works well, @bahl24 can you implement "reset to default" ?

**Review by MauricioFauth — APPROVED** [review]

**Review by MauricioFauth — COMMENTED** [review]
I think it's better to just change the font size of the query text, not the entire console box. What do you think @williamdes.

**Comment by williamdes** [comment]
@mauriciofauth yes, I agree

**Pull request closed without merging**